index.html

```diff
@@ -8,7 +8,7 @@
 	<h1>Cool Generator</h1>
 	<input class="color1" type="color" name="color1" value="#00ff00">
 	<input class="color2" type="color" name="color2" value="#ff0000">
-	<h2>Current CSS Background</h2>
+	<h2>This is the Background</h2>
```

Comment:
looks good, but maybe this is better:
```suggestion
	<h2>This is the Background!</h2>
```